xclim/data/icclim.yml

```diff
@@ -62,6 +62,7 @@ indices:
       long_name: Biologically effective growing degree days (Summation of min((max((Tmin + Tmax)/2 - {thresh_tasmin}, 0)), 9°C), for days between 1 April and 30 September)
       comment: Revised formula published by ECAD/KNMI for ICCLIM, 2013.
     index_function:
+      name: biologically_effective_degree_days
```

Comment:
I agree... Yours would have work without my suggestion of injecting the `method` parameters in the xclim indicator! Because of that we had to go back to the indice and inject everything... Also first time I inject "null" (None), happy that it works!